Favor synthetics-demo repo over synthetics/examples. (#1108)
elastic/observability-docs: docs/en/observability/synthetics-quickstart.asciidoc

  id: todos
  type: browser
  schedule: "@every 1m"
  params:
    url: "https://elastic.github.io/synthetics-demo/" <3>
  source:
    zip_url: <4>
      url: "https://github.com/elastic/synthetics-demo/archive/refs/heads/main.zip" <5>
      folder: ""todos/synthetics-tests" <6>
      username: "" <7>
      password: "" 
----
<1> Each `monitor` gets its own ID in the {uptime-app} and, therefore its own schedule entry.
This allows tests to be run in parallel and analyzed separately.
<2> In this example, a synthetic test is defined inline. This is a two-step script that first loads
a homepage and then hovers over a product menu. See <<synthetics-syntax>> for more information.

> <3> The params section lets you define custom parameters to be used as you see fit in your script. In this example the 'url' parameter is used by the test suite to determine which site to test.

Reviewer: ```suggestion
<3> The params section lets you define custom parameters to use in your script. In this example the 'url' parameter is used by the test suite to determine which site to test.
```

Reviewer: IMO "custom parameters" implies that phrase "as you see fit"
